### DISCUSSION TITLE
Download .zip from releases

### DISCUSSION THREAD

**Discussion by jamesshenry** [devlooped/dotnet-file#131] [General]
How would I use this to download a .zip file for a github release? Using `dotnet-file add https://github.com/ankitpokhrel/jira-cli/releases/download/v1.5.2/[jira_1.5.2_windows_x86_64.zip` downloads the repository when I would expect it to download just the zip file

**Comment by kzu** [comment]
hm. that's an interesting use case :). Since the tool already uses the GH CLI, it could use the CLI to download the release binaries 🤔 

Moving this to an issue in the hopes it will get traction and (especially) backing via sponsors
